<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DriverScanClass</class>
 <widget class="QMainWindow" name="DriverScanClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DriverScan</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="txtMsg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>581</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btScan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="text">
     <string>扫描</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxItem">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="driverscan.qrc"/>
 </resources>
 <connections/>
</ui>
